playbooks/common/openshift-cluster/config.yml

```diff
@@ -17,6 +17,24 @@
       - package_version
       - docker_image_availability
       - docker_storage
+    when: ansible_distribution != "Fedora"
+
+- name: Verify Requirements - Fedora
+  hosts: oo_all_hosts
+  roles:
+  - openshift_health_checker
+  vars:
+  - r_openshift_health_checker_playbook_context: install
+  post_tasks:
+  - action: openshift_health_check
+    args:
+      checks:
+      - disk_availability
+      - memory_availability
+      - package_availability
+      - docker_image_availability
+      - docker_storage
+    when: ansible_distribution != "Fedora"
```

Comment:
I agree that it isn't too helpful to run checks for supportable resources on Fedora. It would be nice to have the package requirements correct for Fedora but if we're not going to commit to keeping up with that (and I don't feel like we will yet) then best just to leave `package_availability` out too (I think it already exempts itself in some cases?).

I think there is some value in applying the `docker_image_availability` check on Fedora, as that will let you know if your install will end up trying to use images that don't exist, and that seems pretty relevant to me.

There's no particular need to have separate plays here. So maybe:

```
 - name: Verify Install Requirements
   hosts: oo_all_hosts
   roles:
   - openshift_health_checker
   vars:
   - r_openshift_health_checker_playbook_context: install
   post_tasks:

   - name: Verify Requirements - EL
     when: ansible_distribution != "Fedora"
     action: openshift_health_check
     args:
       checks:
       - disk_availability
       - memory_availability
       - package_availability
       - package_version
       - docker_image_availability
       - docker_storage

   - name: Verify Requirements - Fedora
     when: ansible_distribution == "Fedora"
     action: openshift_health_check
     args:
       checks:
       - docker_image_availability
```